src/model/CarManager.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+import Car from './Car.js';
+import pickRandomNumber from '../utils/pickRandomNumber.js';
+import OutputView from '../views/OutputView.js';
+import { CONFIG } from '../constants/config.js';
+
+class CarManager {
+  constructor() {
+    this.cars = [];
+  }
+
+  createCars(carNames) {
+    this.cars = carNames.map((carName) => new Car(carName));
+    return this.cars;
+  }
+
+  isMoveCondition(pickedRandomNumber) {
+    return pickedRandomNumber >= CONFIG.MINIMUM_RANDOM_NUMBER
+    && pickedRandomNumber <= CONFIG.MAXIMUM_RANDOM_NUMBER;
+  }
+
+  moveForwardCar(car, pickedRandomNumber) {
+    const moveCondition = this.isMoveCondition(pickedRandomNumber);
+    if (moveCondition) {
+      return car.move();
```

Comment:
```suggestion
      car.position ++;
```
현재는 Car 클래스 객체의 필드가 public이어서 이렇게도 가능합니다!
그래서 Car 클래스와 move 메서드의 필요성이 잘 안느껴졌어요.

position을 캡슐화 하고 moveCondition 로직을 move 안으로 옮기는 방법으로 수정하시거나,
아예 Car는 클래스로 관리하지 않는 방법은 어떤가요?

Reply:
맞네요. `position`이 public으로 선언되어서 외부에서  `car.position ++`로도 작성할 수 있을 것 같습니다.
하지만 퍼블릭 혹은 프라이빗으로 선언된 것과는 별개로 자동차를 진짜 움직이는 것을 Car 클래스의 역할이라고 생각했습니다.
CarManager 클래스는 자동차들을 관리하는 클래스이고, Car는 각 자동차를 관리하는 클래스라서요. 어떻게 생각하시나요?


아래 부분은 STEP2에서 시도해보겠습니다!
```
position을 캡슐화 하고 moveCondition 로직을 move 안으로 옮기는 방법으로 수정하시거나,
아예 Car는 클래스로 관리하지 않는 방법은 어떤가요?
```

Reply:
정확하게 이해해 주셨네요! 좋습니다 👍 

---

테스트가 없더라도, 우승자 계산이나 출력을 위해서는 값을 가져오는 함수가 필요하긴 합니다! getPosition메서드를 추가하셔야 합니다.
다만 누군가 car.position 을 직접 수정할 수 없게 하는 것으로도 Car 클래스의 역할이 부여된다고 생각해요.